change/@fluentui-react-combobox-64f05178-7fe6-4266-9707-195d14561a0a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30733&commitId=d294d44209a783bd57629c8a80a2ade4278f77b5&env=prod&clientType=FLUENTUI)


  <details><summary><b>Combobox Converged</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Combobox Converged.Appearance- underline.hover.chromium.png|637| Changed|
|Combobox Converged.Appearance- underline.default.chromium.png|637| Changed|
|Combobox Converged.Invalid- underline.hover.chromium.png|632| Changed|
|Combobox Converged.Appearance- underline.focused.chromium.png|652| Changed|
|Combobox Converged.Invalid- underline.default.chromium.png|632| Changed|
|Combobox Converged.Invalid- underline.focused.chromium.png|652| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
It looks like this change increases the height of the underline appearance Combobox to 32px, which is the same as all other appearances. I think that is actually fixing a bug: all comboboxes should be 32px regardless of appearance.